datastore/tests/unit/datastore_test.py

```diff
@@ -1,5 +1,15 @@
-import gcloud.aio.datastore.datastore as datastore  # pylint: disable=unused-import
+import gcloud.aio.datastore.datastore as datastore
+from gcloud.aio.datastore.key import Key
+from gcloud.aio.datastore.value import Value
 
 
-def test_importable():
-    assert True
+def test_defining_custom_subclass_updates_value_key_kind():
+    class CustomKey:
+        pass
+
+    class CustomDatastore(datastore.Datastore):  # pylint: disable=unused-variable
+        key_kind = CustomKey
+
+    assert Value.key_kind == CustomKey
+
+    Value.key_kind = Key  # Teardown
```

Comment:
The fact that this line is necessary highlights the fact that you can shoot yourself in the foot if you're using more than one Datastore class since `Value.key_kind` will take the `key_kind` of whichever was defined last (even if it's not used at all).

Without this line, `Value.key_kind` stays set to `CustomKey` for whatever tests you are running next. So if you run the Value tests after this one, some of them will fail.

I think it's an edge case that demands a lot of attention so my preference would be to open a ticket to take care of that later. I expect the normal workflow to define your own Datastore class (or use the default one) and set `key_kind` there once and for all.

Reply:
Yes, that's a great thought to call out -- I agree this is an important edge case. Could you please open a GitHub issue on this repo so we don't forget to look into this?

Reply:
Okay, I'll see what this edge case looks like when I've made the other changes, and create an issue